crates/hc_bundle/src/packing/wasmer_sys.rs

```diff
@@ -1,5 +1,3 @@
-#![cfg(feature = "wasmer_sys")]
```

Comment:
Was specified on the `mod xxx` declaration and inside the module, Clippy now spots that one is redundant.